_shows/75_76/smoking_is_bad_for_you.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Smoking is Bad for You
+playwright: Anton Checkov
```

Comment:
There is a different spelling of Chekhov in the summary to here. (Chekhov is correct)